src/main/java/io/stargate/sgv2/jsonapi/api/configuration/CommandObjectMapperHandler.java

```diff
@@ -38,7 +38,7 @@ public boolean handleUnknownProperty(
     }
     if (typeStr.endsWith("CreateCollectionCommand$Options$VectorSearchConfig")) {
       throw ErrorCodeV1.INVALID_CREATE_COLLECTION_OPTIONS.toApiException(
-          "Unrecognized field \"%s\" for `createCollection.options.vector` (known fields: \"dimension\", \"metric\", \"service\")",
+          "Unrecognized field \"%s\" for `createCollection.options.vector` (known fields: \"dimension\", \"metric\", \"sourceModel\", \"service\")",
```

Comment:
nit: keep property names in alphabetic order, so `... \"service\", \"sourceModel\")"`